src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,54 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val frenchGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut tout le monde"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Salut $someone"
+            }
+        }
+
+        val spanishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hola, mundo"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hola, $someone"
+            }
+        }
```

Comment:
let's write it more concisely like this
```suggestion
        val englishGreeting = object : HelloWorldGreeting {
            override fun greet(): String = "Hello world"
            override fun greetSomeone(someone: String): String = "Hello $someone"
        }

        val frenchGreeting = object : HelloWorldGreeting {
            override fun greet(): String = "Salut tout le monde"
            override fun greetSomeone(someone: String): String = "Salut $someone"
        }

        val spanishGreeting = object : HelloWorldGreeting {
            override fun greet(): String = "Hola, mundo"
            override fun greetSomeone(someone: String): String = "Hola, $someone"
        }```